src/validator.ts

```diff
@@ -38,7 +39,13 @@ export async function validateModel(
 export async function parseRaml(
   filename: string
 ): Promise<amf.model.document.BaseUnit> {
-  const parser = new Raml10Parser();
+  const ccStandardResourceLoader = new FatRamlResourceLoader(
```

Comment:
Refactors the class name but then this instantiates it as ccStandardResourceLoader.  I would prefer this to also be called something like fatRamlResourceLoader instead.